styles.css

```diff
@@ -144,6 +148,7 @@ button {
 .search-button {
   border: none;
   width: 5vw;
+  height: 4.9vw;
```

Comment:
nice job with paying attention to some of these visual details and like that you went with vw